firmware/tools/make_baseband_file.py

```diff
@@ -114,4 +114,4 @@ def write_file(data, path):
 	# Update original binary with MD5 footprint
 	write_file(data[512:(32768+512)], args + '/build/' + args + '_inc.bin')
 
-write_file(h_data, 'common/modules.h')
+# write_file(h_data, 'common/modules.h')
```

Comment:
why keep it at all ?

Reply:
this changes the bin files and don't know what this py does, so just removed the file write part.
if someone know what part can be removed, please do it

Reply:
hmm maybe @bernd-herzog have an idea ?

Reply:
nope, no idea what that does

Reply:
After a huge check, that file is not used anywhere anymore. Let's kill it.